dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -86,15 +93,18 @@ $ oc set probe dc/nexus \
 ====
 If you do not want persistent storage, continue to
 xref:nexus-connecting-to-nexus[Connecting to Nexus]. However, your cached
-dependencies will be lost if the pod is restarted for any reason.
+dependencies and any configuration customization will be lost if the pod is
+restarted for any reason.
 ====
 
 Create a persistent volume claim (PVC) for Nexus, so that your dependencies
-will not be lost when the pod running the server terminates. If you do not have
-administrator access on your cluster, ask your system administrator to create a
-Read/Write Persistent Volume for you.  Otherwise, read
-xref:../volumes.adoc#dev-guide-volumes[Managing Volumes in {product-title}] for instructions on
-creating a persistent volume.
+will not be lost when the pod running the server terminates. PVCs require
+available persistent volumes (PV) in your cluster. If there are no PVs
+available in your cluster and you do not have administrator access on your
+cluster, ask your system administrator to create a Read/Write Persistent Volume
+for you.  Otherwise, read
+xref:../volumes.adoc#dev-guide-volumes[Managing Volumes in {product-title}] for
+instructions on creating a persistent volume.
 
 Add a PVC to the Nexus deployment configuration.
```

Comment:
I think @csrwng suggested you can use --replace instead of --remove and --add in the instructions below, though i'm not seeing that option in oc help right now.  Double check with him.